src/vs/workbench/electron-browser/bootstrap/index.js

```diff
@@ -81,6 +81,30 @@ function readFile(file) {
 	});
 }
 
+function showPartsSplash(configuration) {
+
+	let key;
+	let keep = false;
+	if (configuration.folderPath) {
+		key = `storage://workspace/${configuration.folderPath.replace(/^\//, '')}/parts-splash`;
+	} else if (configuration.workspace) {
+		key = `storage://workspace/root:${configuration.workspace.id}/parts-splash`;
+	} else {
+		key = `storage://global/parts-splash`;
+		keep = true;
+	}
+
+	let structure = window.localStorage.getItem(key);
```

Comment:
This is now the new *first* access to local storage. I do recall that we have some special perf-ticks for that but I am unsure if that is still the case and iff so where that is.